Fix for issue #10

ngrigoriev opened the pull request:
Check the index status report for the new indexes to make sure they get
registered correctly.

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
Fix for issue #10

Check the index status report for the new indexes to make sure they get
registered correctly.

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
Fix for issue #10

Check the index status report for the new indexes to make sure they get
registered correctly.

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

vicneanschi commented on src/main/java/com/newforma/titan/schema/SchemaManager.java:
```diff
@@ -878,18 +879,23 @@ private void populateNewIndexes(JanusGraph graph, GraphState graphState) throws
-				LOG.info("Index {} is enabled, existing data may need to be reindexed", indexName);
-
-				graphState.addIndex(indexName, true);
+				if (indexStatusReport.getSucceeded()) {
```
align with closing bracket

vicneanschi commented on src/main/java/com/newforma/titan/schema/SchemaManager.java:
```diff
@@ -926,18 +932,23 @@ private void populateNewLocalPropIndexes(JanusGraph graph, GraphState graphState
-				mgmtIndexEnabler.commit();
-				LOG.info("Local property index {} is enabled, existing data may need to be reindexed", indexName);
+				if (indexStatusReport.getSucceeded()) {
+    				LOG.info("Enabling local property index {}...", indexName);
```
indent "if" body

vicneanschi commented on src/main/java/com/newforma/titan/schema/SchemaManager.java:
```diff
@@ -975,18 +986,23 @@ private void populateNewLocalEdgeIndexes(JanusGraph graph, GraphState graphState
-				LOG.info("Local edge index {} is enabled, existing data may need to be reindexed", indexName);
-
-				graphState.addIndex(indexName, true);
+				if (indexStatusReport.getSucceeded()) {
```
fix indentation

ngrigoriev committed:
Fix for issue #21 (#24)

Add long option names, add constant for "g" option, uncomment previously
commented code to enable the saving of the graph data.

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

ngrigoriev commented:
Hrr.....it looks like after my last Eclipse update my formatting settings went to /dev/null! :(

Commit:
cosmetic: address code review comments

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
cosmetic: address code review comments

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
add .editorconfig

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
add .editorconfig

Signed-off-by: Nikolai Grigoriev <ngrigoriev@newforma.com>

Commit:
Merge branch 'issue/10' of github.com:ngrigoriev/janusgraph-schema-manager into issue/10
